src/encoded/static/components/app.js

```diff
@@ -1204,6 +1230,12 @@ class App extends React.Component {
                                     signup={this.signup}
                                 /> :
                             null}
+                            {this.state.accountCreationFailedVisibility ?
+                                <AccountCreationFailedModal
+                                    closeModal={this.closeAccountCreationErrorModal}
+                                    date={(new Date()).toUTCString()}
+                                /> :
+                                null}
```

Comment:
This is an optional style comment, but @forresttanaka and I generally format conditionals like this differently. We generally put `: null}` together on one line which is aligned with the opening `{`. Whether or not you want to put the `:` with `null`, I think `null}` should be aligned with the opening `{`, in particular because line 1232 is formatted that way.

Reply:
Do you mean like this:
                            {this.state.accountCreationFailedVisibility ?
                                <AccountCreationFailedModal
                                    closeModal={this.closeAccountCreationErrorModal}
                                    date={(new Date()).toUTCString()}
                                />
                            : null}

Reply:
Yes. Where to put the `:` is completely up to you, though, I have no strong preference. I do think the indentation should be fixed though.

Reply:
Done.